src/todos.js

```diff
@@ -0,0 +1,37 @@
+let express = require('express');
+let bodyParser = require('body-parser');
+
+let app = express();
+app.use(bodyParser.json());
+
+let todos = [{id: 'jkhsdjkf', content: 'review this code'}];
+
+app.post('/todos', (req, res) => {
+  todos.push({
+    ...req.body,
+    id: Math.random().toString(32).slice(2)
+  });
+  res.sendStatus(201);
+});
+
+app.put('/todos/:id', (req, res) => {
+  todos[Number(req.params.id)] = req.body;
```

Comment:
1. Validate id exists, return `404` if not. Use strings as per my comment at L7.
2. Validate request body, ensure to store only expected data.